src/init/features/dataconnect/sdk.ts

```diff
@@ -141,6 +141,7 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
     const javascriptSdk: JavascriptSDK = {
       outputDir,
       package: pkg,
+      packageJsonDir: newConnectorYaml.generate.javascriptSdk?.packageJsonDir,
     };
 
     if (
```

Comment:
I think below we should prompt even if there isn't a package Json dir

Reply:
Feel free to tweak it.

If there is no `package.json`, what would `packageJsonDir` do? Try to `npm i`, but fails?